fileidentification/filehandling.py

```diff
@@ -301,6 +312,9 @@ def run(
         # generate policies
         self._manage_policies(policies_path, blank, extend)
         # probing the files
+        if report_only:
+            self.mode.VERBOSE = True
```

Comment:
I would keep these 2 separated. If there's no strong reason to mix things together, I would rather keep them separated.

Because in my opinion, `--report-only` is another concern than `--verbose`:

- `--verbose`: define how technical / wordy the output is. This depends on the technical level of the user. For newcomers (e.g. typical Galaxy users), `--verbose` would be overwhelming.
- `--report-only`: define if the program should only read, or also write. This doesn't depend on the technical level of the user, but rather on the use case.

```suggestion
```